spec/knackhq/client_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+
+describe Knackhq::Client do
+  it 'has a version number' do
+    expect(Knackhq::Client::VERSION).not_to be nil
+  end
+
+  let(:client) { knack_client }
+  let(:base_uri) { 'http://example.com' }
+  let(:x_knack_application_id) { '00000000' }
+  let(:x_knack_rest_api_key) { '99999999' }
+
+  let(:knack_client) do
+    Knackhq::Client.new(base_uri,
+                        x_knack_application_id,
+                        x_knack_rest_api_key)
+  end
+
+  describe '#new' do
+    it { expect(knack_client).to be_a Knackhq::Client }
+  end
+
+  describe '#request.get' do
```

Comment:
I would suggest that the expected Blanket types/instance are returned, the method is wrapping blanket functionality so all we need to know is that it produces the expected Blanket results.

Personally I probably wouldn't expose the blanket request, I would nail the client down more so that arbitrary requests could not be made. If there was (and I imagine with Knack there is) a need to make more dynamic requests I may implement as:

```ruby
def object(:key)
  # request is private
  # I don't know if this is a valid blanket request, but its for demonstration purposes.
  request.get.objects(:key)
end
```

Thoughts?